src/index.html

```diff
@@ -9,7 +9,6 @@
 
 <body>
 	<div id="root" style="height: 100%"></div>
-	<script src="./app/main.js"></script>
```

Comment:
you dont need this as htmlWebpackPlugin takes care of inserting bundled `.js`files